go/api/requests/document.go

```diff
@@ -0,0 +1,9 @@
+package requests
+
+import "mime/multipart"
+
+type DocumentCreateBody struct {
+	Title       string                `json:"title" binding:"required"`
+	Description string                `json:"description" binding:"required"`
```

Comment:
```suggestion
	Description string                `json:"description"`
```
Je suis pas sur pour le binding:"required" sur la description ?

Je pense pas qu'avoir une description sur un passport serve à grand chose par exemple